hw/top_earlgrey/dv/verilator/chip_sim_tb.cc

```diff
@@ -26,11 +28,17 @@ int main(int argc, char **argv) {
               0x4000 / 4, 4);
   MemArea ram(top_scope + ".u_ram1p_ram_main." + ram1p_adv_scope, 0x20000 / 4,
               4);
+  // Only handle the lower bank of flash for now.
   MemArea flash(top_scope +
                     ".u_flash_ctrl.u_eflash.u_flash.gen_generic.u_impl_generic."
                     "gen_prim_flash_banks[0].u_prim_flash_bank.u_mem."
                     "gen_generic.u_impl_generic",
-                0x100000 / 8, 8);
+                0x80000 / 8, 8);
```

Comment:
nit: I haven't looked if it is possible, but is there a way to bring in the autogenerated constants? Not sure if worth the effort in this PR.

Reply:
Hm... I'm not sure. If we want all the necessary information, we would need flash_ctrl_regs.h (for the number of banks or bank size), but that's generated via bazel. We would likely need to then plug that file location into a core file so fusesoc can pick it up.

I think it might not be worth it for this PR. At some point, we probably want to remove fusesoc from the build system, so we can track files and dependencies in bazel without there being yet another spec in core files.